src/data/disks.rs

```diff
@@ -78,16 +78,23 @@ impl Disk {
 /// Wrapper above the `sysinfo::Disks` struct
 #[derive(Default, Debug)]
 pub struct Disks {
-    disks: Vec<Disk>,
+    pub disks: Vec<Disk>,
+}
+
+pub fn native_load_disks(canonicalize_paths: bool) -> Disks {
+    Disks { disks: load_disks(canonicalize_paths) }
 }
```

Comment:
This should be in the `impl Disks` block and best renamed to `new_native_disks` or something similar